sklearn/svm/_classes.py

```diff
@@ -1155,6 +1155,8 @@ class SVR(RegressorMixin, BaseLibSVM):
     other :ref:`kernel_approximation`.
 
     Read more in the :ref:`User Guide <svm_regression>`.
+    See :ref:`sphx_glr_auto_examples_svm_plot_svm_regression.py`
+    for a regression example using SVR with different kernels.
```

Comment:
Could you move this to `kernel` under the `Parameters` section of the `SVM` class? You can refer to how it’s done in the `SVC` class; the case of `NuSVM` is similar.

<img width="458" alt="svc_kernels" src="https://github.com/user-attachments/assets/2f08a507-72aa-4215-8e86-6b06ae541eac">